sentry_sdk/integrations/opentelemetry/scope.py

```diff
@@ -123,7 +123,7 @@ def _incoming_otel_span_context(self):
         # for twp to work, we also need to consider deferred sampling when the sampling
         # flag is not present, so the above TraceFlags are not sufficient
         if self._propagation_context.parent_sampled is None:
-            trace_state = trace_state.add(TRACESTATE_SAMPLED_KEY, "deferred")
+            trace_state = trace_state.update(TRACESTATE_SAMPLED_KEY, "deferred")
```

Comment:
Added this change, to make sure we set sentry-sampled to deferred in any case. If there is already a entry sentry-sampled in the trace_state then .add() will not do anything. 

(There can be a sentry-sampled in the trace_state if the upstream SDK sends the `sentry-sampled` DSC entry with the `baggage` http request. If there was no upstream sampling descision, this entry should either not be sent, of if sent than as `None`.)

This change works in both cases.